src/util/text.rs

```diff
@@ -212,6 +212,13 @@ pub(crate) fn encode_text(text: &str, text_encoding: TextEncoding, terminated: b
 	}
 }
 
+pub(crate) fn trim_end_nulls(text: &mut String) {
```

Comment:
Why `trim_end...` vs. `trim_trailing_...`? I would prefer a consistent naming of these functions.

Reply:
I chose `trim_end_nulls` to be consistent with `trim_end_matches` from `std::str`. I guess I used `trim_trailing_...` in the test name because I thought it was a bit more descriptive. Should I change the test name to `trim_end...`?

Reply:
I didn't notice that this only affects the test code. Then `trim_end_nulls` is fine here. If you deliberately named the test then leave it as it.

Reply:
...though renaming the test would be helpful, if I think about it again. Easier to correlate.